### PR TITLE
Rename CacheEntry to EntryHandle.

### PR DESCRIPTION
**Pull request by eolivelli**
Since new 2.0 API needs an explicit release of entries it is better to break binary compatibility

### CODE REVIEW

File: blazingcache-core/src/main/java/blazingcache/client/EntryHandle.java [hunk 1]
```diff
@@ -28,17 +28,18 @@
...
+ * An handle to an entry.
+ * <p>
+ * Entries are reference counted so you have to call explicitly {@link #close() }
+ * in order to release the resources associated to the entry.
  *
  * @author enrico.olivelli
  */
 @SuppressFBWarnings({"EI_EXPOSE_REP2", "EI_EXPOSE_REP"})
-public final class CacheEntry implements AutoCloseable {
+public final class EntryHandle implements AutoCloseable {
```
Comment (diegosalvi) [inline blazingcache-core/src/main/java/blazingcache/client/EntryHandle.java]: Just to be cleaner toString method too should be changed

Comment (eolivelli) [reply blazingcache-core/src/main/java/blazingcache/client/EntryHandle.java]: good catch.
resovled

**Comment by diegosalvi** [comment]
All good, waiting CI

**Comment by eolivelli** [comment]
@diegosalvi  CI is green thanks for your review

**Pull request merged**